src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,13 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private static final UserService userService = new UserService();
+
+    public AuthenticationService() {
+    }
```

Comment:
why did you added this? java classes always has a default constructor, if not implemented others
```suggestion
```

Reply:
I made all corrections